### PR TITLE
docs - add GUC -write_to_gpfdist_timeout

### PR DESCRIPTION
**Pull request by mkiyama**
Add GUC and add link to GUC in gpfdist reference

Related dev PR #10214

Will be backported to 4.3.x

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 1]
```diff
@@ -9413,6 +9415,37 @@ statement_mem = rg_perseg_mem / max_expected_concurrent_queries</codeblock></li>
       </table>
     </body>
   </topic>
+  <topic id="write_to_gpfdist_timeout">
+    <title>write_to_gpfdist_timeout</title>
+    <body>
+      <p>Controls the time (in seconds) that Greenplum Database waits before returning an error when
+        Greenplum Database is attempting to write to a <codeph><xref
+            href="../../utility_guide/ref/gpfdist.xml"/></codeph> server and
+          <codeph>gpfdist</codeph> does not respond. The default value is 600 (10 minutes). A value
```
Comment (huiliang-liu) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml:9424]: default value is 300 (5 minutes)

File: gpdb-doc/dita/utility_guide/ref/gpfdist.xml [hunk 2]
```diff
@@ -213,6 +213,11 @@
         </ul>
         <note type="warning">Disabling SSL certificate authentication exposes a security risk by not
           validating the <codeph>gpfdists</codeph> SSL certificate. </note></p>
+      <p>You can set the server configuration parameter <codeph><xref
+            href="../../ref_guide/config_params/guc-list.xml#write_to_gpfdist_timeout"/></codeph> to
+        control the time that Greenplum Database waits before returning an error when a
+          <codeph>gpfdist</codeph> server does not respond while Greenplum Database is attempting to
+        write data to <codeph>gpfdist</codeph>. The default is 600 seconds (10 minutes).</p>
```
Comment (huiliang-liu) [inline gpdb-doc/dita/utility_guide/ref/gpfdist.xml:220]: same as above

**Comment by lij55** [comment]
Hi @mkiyama , thanks for the doc.
The key point is that gpfdist will return immediately without any kind of retry if there is any error occur during this timeout (and the timeout is less than 600s). It is what customers care the most.

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml (hunk 1 above)
Comment (mkiyama) [reply gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: Thanks. Updated default.

File: gpdb-doc/dita/utility_guide/ref/gpfdist.xml (hunk 2 above)
Comment (mkiyama) [reply gpdb-doc/dita/utility_guide/ref/gpfdist.xml]: fixed

**Pull request merged**